README.md

```diff
@@ -216,13 +222,15 @@ A file with only the first section can be auto-generated in some cases by
 above). Parts of the second section can be added and adjusted in stages when
 you wish to customize the behavior of `zulip-term`.
 
+If you’re downloading the config file from your Zulip account, you should replace the `key` field with `passcmd`, setting its value to a command that outputs the api_key (e.g., cat zulip_key). If you’re not downloading it manually, zulip-term will configure this for you automatically, though it’s recommended to update the passcmd value afterward for better security.
+
 The example below, with dummy `[api]` section contents, represents a working
 configuration file with all the default compatible `[zterm]` values uncommented
 and with accompanying notes:
 ```
 [api]
 email=example@example.com
-key=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+passcmd=cat zulip_key
```

Comment:
Since this needs to be optional, and if in Terminal only will be specific to us, it needs to be briefly documented in the sample file, and commented out - see how we document the Terminal options further below.